e2e/nomostest/prometheus_metrics.go

```diff
@@ -46,22 +46,26 @@ type MetricsPredicate func(context.Context, prometheusv1.API) error
 // until successful (with timeout).
 func ValidateMetrics(nt *NT, predicates ...MetricsPredicate) error {
 	ctx, cancel := context.WithCancel(nt.Context)
-	defer cancel() // stop port-forward
-	address, err := portForwardPrometheus(ctx, nt)
-	if err != nil {
-		return err
-	}
-	client, err := prometheusapi.NewClient(prometheusapi.Config{
-		Address: fmt.Sprintf("http://%s", address),
-	})
-	if err != nil {
-		return err
-	}
-	v1api := prometheusv1.NewAPI(client)
+	defer cancel()
+	var address string
+	var client prometheusapi.Client
```

Comment:
the client doesn't need to be exposed outside of the retry, just the v1api that wraps i.

Reply:
Done